contracts/DNSSECImpl.sol

```diff
@@ -63,7 +63,7 @@ contract DNSSECImpl is DNSSEC, Owned {
         anchors = _anchors;
         rrsets[keccak256(hex"00")][DNSTYPE_DS] = RRSet({
             inception: uint32(0),
-            inserted: uint64(block.timestamp),
+            expiration: uint32(3767581600), // May 22 2089 - the latest date we can encode as of writing this
```

Comment:
Are we going to have Y2089 problem? I am highly unlikely to work till that year but are people in future have to worry about it?

Reply:
Hopefully the anchors will have been replaced again by that point, in which case this will be replaced by the new ones.